Sources/JavaScriptKit/Support.swift

```diff
@@ -0,0 +1,43 @@
+protocol _AnyJSValueConvertible: JSValueConvertible {}
+
+/// A type-erased wrapper for types conforming to JSValueConvertible
+public struct AnyJSValueConvertible: JSValueCodable, ExpressibleByNilLiteral {
```

Comment:
<img width="741" alt="Screenshot_2020-09-15 17 55 35" src="https://user-images.githubusercontent.com/25517624/93269297-b604e180-f77c-11ea-8e27-e7c2585325c0.png">

Reply:
In that case, users should use JSValue instead of `JSValueCodable`.

Reply:
I’ve found one more place `AnyJSValueConvertible` comes up: APIs like [`console`](https://github.com/swiftwasm/DOMKit/blob/main/Sources/DOMKit/WebIDL/console.swift) that accept any value. I think it is possible to get this to work but it would require a redesign of a bunch of generated code.

Reply:
Hmm, I think the arguments of `console` methods can be replaced with `JSValue` also

Reply:
Yep! There was just one case with an `any` in the WebIDL files that had a default `nil` value which should be cleared up by #59.